R/run_prep_calculation_loans.R

```diff
@@ -98,6 +98,10 @@ run_prep_calculation_loans <- function(input_path_project_specific,
       sheet = "Company Indicators - PACTA Comp"
```

Comment:
In general: what are the expected changes on the results of loanbook prep? Did you check that there is no unintended impact on trisk (I would assume there is if we lose rows as indicated in one comment..)

Reply:
After fixing the issue of why rows were dropped when adding emission factors, there are no further differences to results when running trisk on master versus this branch

Reply:
amazing